### PR TITLE
Optimize a few things

### PR DESCRIPTION
**Pull request by thetwopct**
Realised this repo could do with some love 😢 ❤️ 

1. Alphabetizised the projects in the readme and in each level index.
2. Moved projects between levels indexes where they have progressed.
3. Kube OVN listing was missing so has been re-added.
4. Fluentbit is not a project and removed from Graduated level index (sub-project of fluent)
5. Updated some file names to be consistent (will be continued after a-z is merged)
6. Ran some SVGs through autocrop as formatting was off (will be continued after a-z is merged)

cc @cjyabraham

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,167 +7,166 @@ With many browsers, you can right click and copy the logo of your choice below.
...
   * [KCL](examples/sandbox.md#kcl-logos)
   * [KCP](examples/sandbox.md#kcp-logos)
-  * [Curve](examples/sandbox.md#curve-logos)
-  * [externl-secrets](examples/sandbox.md#external-secrets-operator-logos)
+  * [Kepler](examples/sandbox.md#kepler-logos)
+  * [Keylime](examples/sandbox.md#keylime-logos)
+  * [Konveyor](examples/sandbox.md#konveyor-logos)
+  * [Krator](examples/sandbox.md#krator-logos)
   * [Krustlet](examples/sandbox.md#ckrustlet-logos)
+  * ~~[Kube-OVN](examples/sandbox.md#kube-ovn-logos)~~
```
Comment (cjyabraham) [inline README.md:112]: Why is this strikethrough?

**Review by cjyabraham — CHANGES_REQUESTED** [review]
Looks like Archived Projects listing in the README is missing [a few](https://www.cncf.io/archived-projects/)?

For the white logos, could we use a darker grey background to help them stand out more?

The [Akri stacked logos](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#akri-logos) are broken.

Why are the [Devfile logos](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#devfile-logos) so small?

Can you make sure the white logos go on a grey background? [These ones are problematic](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#merbridge-logos) and there are several others. Maybe we should just use a grey background throughout to keep it simpler?

[Open Service Mesh logos are broken](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#open-service-mesh-logos).

[Service Mesh Interface Logos](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#service-mesh-interface-logos) are broken.

File: README.md (hunk 1 above)
Comment (thetwopct) [reply README.md:112]: It was missing, but I re-added it.

**Comment by thetwopct** [comment]
@cjyabraham 

1) Out of scope. I didn't compare data on that level; if the logos are missing, the logos likely don't exist in the repo. As I work through the whole repo this will become clearer. This is part 1 of possibly many parts. Wasn't expecting to need to completely rejig the MD files, but it's essential to do that first because otherwise any other PRs will have issues merging. Welcome to compare and PR if needed.
2) Out of scope. Thats not something I will be covering in this optimisation. But the MD files don't support `<style>` tag, so it would need to be inline styles on every table. PRs welcome.
3) Done. Had renamed them and missed "stack" changing to "stacked"
4) Done. This PR only includes updates to about 5 folders of images out of about 250 folder updates required. But have fixed this for you by running it through Autocrop now instead of later.
5) On the list. Merbridge goes outside the conventions of color + black + white layout of all other logos so this logo pack needs to be updated and the table needs to be updated. There are maybe 5-10 projects that need fixing in this way.
6) Done. It's been archived. Removed and updated archived.md
7) Done. It's been archived. Removed and updated archived.md

**Comment by edsiper** [comment]
@thetwopct why Fluent Bit has been removed without asking Fluent maintainers about this ?

Fluent Bit is a graduated project under the umbrella of Fluentd, this has been approved while ago by CNCF.

**Comment by thetwopct** [comment]
@edsiper ok. Is there a reason why the project is not listed on the Projects page under Graduated? 

https://www.cncf.io/projects/

**Comment by edsiper** [comment]
sub projects are not listed in the principal page

**Comment by thetwopct** [comment]
@edsiper Hey Eduardo, we have [restored the fluentbit artwork to the repo](https://github.com/cncf/artwork/blob/main/examples/graduated.md#fluentbit-logos) and improved how we handle the display of subprojects so hopefully this should be clear going forward. Thanks for bringing this to our attention.

**Comment by edsiper** [comment]
@thetwopct thank you !

**Pull request merged**